### PR TITLE
97 client dashboard front and therapist dashboard frontend

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#97, #103
how to test: for now the authentication not completed yet, so you need to pass the user type to the navbar 
on `client\src\components\Navbar\index.js`
line 25 `const NavRoutes = decideRoutes(user.role);`
pass the user type you want to test here 
EX: `const NavRoutes = decideRoutes('THERAPIST');`
EX: `  const NavRoutes = decideRoutes('CLIENT');`

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/pages/Therapist/Dashboard/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      <Row>
+        <Col w={[4, 12, 12]}>
+          <T.H2>
+            Welcome back, <S.BoldSpan> {data.firstName} </S.BoldSpan>!
+          </T.H2>
+        </Col>
+      </Row>
+      <Row>
+        <Col w={[4, 12, 12]} mt={7} mtT={5}>
+          <T.H3>My Clients</T.H3>
```
Comment (thejoefriel) [inline client/src/pages/Therapist/Dashboard/index.js:39]: should be lowercase c so "My clients"

File: client/src/pages/Therapist/Dashboard/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    programme: { date: moment(), id: 0 },
+  };
+
+  return (
+    <div style={{ maxWidth: 1068 }}>
+      <Row>
+        <Col w={[4, 12, 12]}>
+          <T.H2>
+            Welcome back, <S.BoldSpan> {data.firstName} </S.BoldSpan>!
```
Comment (thejoefriel) [inline client/src/pages/Therapist/Dashboard/index.js:33]: Remove the extra space after firstName

**Review by thejoefriel — CHANGES_REQUESTED** [review]
- [x] This isn't an error, but can you change "Contact my therapist" to "Contact therapist". This will hopefully make both links in the card stay on the same row within smaller screen widths
- [x] The font weight for "My" should be '300'
image
- [x] There should be 64px spacing at the bottom of the General layout, but doesn't seem to be here (its only 32). Can you increase please 
- [ ] There should be 32px spacing at the to, currently there's 48px. Can you please fix

File: client/src/pages/Therapist/Dashboard/index.js (hunk 1 above)
Comment (fadeomar) [reply client/src/pages/Therapist/Dashboard/index.js:39]: done

File: client/src/pages/Therapist/Dashboard/index.js (hunk 2 above)
Comment (fadeomar) [reply client/src/pages/Therapist/Dashboard/index.js:33]: done

**Review by thejoefriel — APPROVED** [review]

**Pull request merged**